src/vs/workbench/parts/terminal/common/terminal.ts

```diff
@@ -100,6 +100,7 @@ export interface ITerminalConfiguration {
 	};
 	showExitAlert: boolean;
 	experimentalBufferImpl: 'JsArray' | 'TypedArray';
+	splitCwdSource: 'ShellDefault' | 'SourceInitialCwd' | 'Cwd';
```

Comment:
Normally we case settings with lowercase first `shellDefault`, `experimentalBufferImpl` is a bad example as those are the values in xterm.js 😅